packages/application-generic/src/tracing/index.ts

```diff
@@ -0,0 +1,31 @@
+import { OpenTelemetryModule } from 'nestjs-otel';
+export * from './tracing';
+
+export {
+  TraceService,
+  MetricService,
+  OtelInstanceCounter,
+  OtelUpDownCounter,
+  OtelHistogram,
+  OtelObservableGauge,
+  OtelObservableCounter,
+  OtelObservableUpDownCounter,
+  OtelCounter,
+  Span,
+} from 'nestjs-otel';
```

Comment:
Is it possible to create our own wrappers around those functions? so that we can also send metrics to new relic or other custom monitoring tools like Clouwatch in case we would want to?

Reply:
Just as a note, I am not sure if that is needed as both of these monitoring systems support OTEL.
https://aws.amazon.com/about-aws/whats-new/2023/08/amazon-cloudwatch-agent-opentelemetry-traces-x-ray/
https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/

However, at what capacity I am not sure and I think this requires a bit more understanding of the trade-offs and usages.

Reply:
Yes, but from previous test we made with NewRelic is very poor compared to their native APM functionality, I would love to to keep the current functionality and not deprecate it until we are happy with the OTEL metrics we neeed.

Reply:
@Cliftonz @scopsy. Could you take a look at the last changes within packages/application-generic/src/tracing/index.ts and packages/application-generic/src/tracing/otel-wrapper.ts if I'm on right way? I wrapped only `Span`, `TraceService`, `MetricService` and if it's ok I'll add other.